free-1.md

```diff
@@ -9,42 +9,141 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+This language serves the need to take notes in economics classes on the computer.
+My personal experience, and those of a couple other economics students, finds
+that taking notes on the computer allows for a more complete transcript of the
+lessons given by economics professors. Economics professors tend to provide
+powerpoint slides that outline major points -- or worse, major questions -- of
+the lecture and then provide the context or answers to these points verbally.
+Often, the homework questions and exam questions relate to the things _said_ in
+class. Thus, it's extremely helpful to transcribe lectures, in my opinion. 
+Typing on the computer is much faster than writing by hand, so transcribing is
+more possible on the computer.  However, the biggest issue is that economics 
+involves many graphs. Copying down these graphs by hand is currently the main 
```

Comment:
I'm going to assume you want to transcribe simple supply-demand curves and not jagged graphs displaying stock prices (or something like that). Because you can just screenshot or take pictures of complicated graphs.